cg/meta/workflow/raredisease.py

```diff
@@ -29,6 +29,16 @@ def __init__(
     def root(self) -> str:
         return self.config.raredisease.root
 
+    def write_managed_variants(self, case_id: str, content: list[str]) -> None:
```

Comment:
Is it possible (and reasonable) to move this function to `AnalysisAPI` so that is not defined twice?

Reply:
I do not think so since we have to send in the root.